test/functional/test_framework/util.py

```diff
@@ -288,6 +289,13 @@ def connect_nodes_bi(nodes, a, b):
     connect_nodes(nodes[a], b)
     connect_nodes(nodes[b], a)
 
+def sha256d(byte_str):
```

Comment:
Please place this in the 'Utility Functions' section, rather than 'Node Functions'